packages/utilities/psammead-styles/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bbc/psammead-styles",
-  "version": "2.3.0",
+  "version": "2.4.0",
```

Comment:
As this changes the package from exporting strings, to functions, this is now a 'Breaking' change (anyone using the package will break), and needs to bump a 'Major' version (to 3.0.0) :) More info can be found here https://semver.org/